src/test/rustdoc-ui/intra-doc/crate-nonexistent.rs

```diff
@@ -0,0 +1,4 @@
+#![deny(rustdoc::broken_intra_doc_links)]
+
+/// [crate::DoesNotExist] //~ ERROR unresolved link to `crate::DoesNotExist`
```

Comment:
I'd rather the comment not be part of the docs it's testing:

```suggestion
/// [crate::DoesNotExist]
//~^ ERROR unresolved link to `crate::DoesNotExist`
```